### ISSUE TITLE
Contribute in flutter

### ISSUE THREAD

**Issue by wasibhussain** [wasibhussain/CodeForAll-Hacktoberfest#4]
Contribute in Flutter with useful and authenticated code or tips/Documentation.

**Comment by Yaswanth-Mitta** [comment]
hey @wasibhussain  I want to contribute in flutter i made some good looking screens can i add

**Comment by wasibhussain** [comment]
@Yaswanth-Mitta yes sure you can!

**Comment by Yaswanth-Mitta** [comment]
hey @wasibhussain  are you aware to follow this while merging 
![image](https://github.com/wasibhussain/CodeForAll-Hacktoberfest/assets/98637209/c9970888-f119-4b99-805a-2af2b1756a67)

**Comment by wasibhussain** [comment]
@Yaswanth-Mitta yes i'm aware of these.

**Comment by Yaswanth-Mitta** [comment]
@wasibhussain ohh great, Thanks 
I am new to open source and git i was contributing to another project got this error can you help me with that 
![image](https://github.com/wasibhussain/CodeForAll-Hacktoberfest/assets/98637209/3bbfe8cd-2e11-40fc-8d7a-5f9ba5b7312e)

**Comment by wasibhussain** [comment]
@Yaswanth-Mitta did you added commit before pushing changes?

**Comment by Yaswanth-Mitta** [comment]
@wasibhussain  
yeah i did commited  but i didnt find new commits  in commits i did it the same way done your PR

**Comment by wasibhussain** [comment]
@Yaswanth-Mitta try to fork and clone the repo again and then push the changes, i think there is github token issue, maybe it could be resolved by doing this.

**Comment by Yaswanth-Mitta** [comment]
@wasibhussain  yeah bro , it got merged no worries thank you. Can i contribute to same project twice for hacktoberfest?

**Comment by wasibhussain** [comment]
@Yaswanth-Mitta , not sure! but i think yes you can. try it!

**Comment by Yaswanth-Mitta** [comment]
@wasibhussain hacktoberfest is not consider the commit in this repo it maybe because issue must be labeled as hacktoberfest

**Comment by wasibhussain** [comment]
@Yaswanth-Mitta no its just a label, it doesn't effect on your commit, BTW I'm labeling it now.

**Comment by DOSDX19** [comment]
Hi @wasibhussain I want to contribute in flutter could I ?

**Comment by wasibhussain** [comment]
@DOSDX19 yes you can..
